mono/mini/debugger-agent.c

```diff
@@ -7207,6 +7208,21 @@ domain_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
 		buffer_add_objid (buf, (MonoObject*)o);
 		break;
 	}
+	case CMD_APPDOMAIN_CREATE_BYTE_ARRAY: {
+		ERROR_DECL (error);
+		MonoArray *arr;
+		gpointer elem;
+		domain = decode_domainid (p, &p, end, NULL, &err);
+		uintptr_t size = 0;
+		int len = decode_int (p, &p, end);
+		size = len;
+		arr = mono_array_new_full_checked (mono_domain_get (), mono_class_create_array (mono_get_byte_class(), 1), &size, NULL, error);
+		elem = (gpointer*)arr->vector;
+		memcpy(elem, p, len);
```

Comment:
This should be 
elem = mono_array_addr_internal (arr, guint8, 0);

Reply:
Space after `memcpy`.
No need for cast, or `elem` temporary.